mathics/core/expression.py

```diff
@@ -1406,8 +1454,13 @@ def set_element(self, index: int, value):
     def shallow_copy(self) -> "Expression":
         # this is a minimal, shallow copy: head, elements are shared with
         # the original, only the Expression instance is new.
+
+        # FIXME: this should be encapulated in the constructor better.
         expr = Expression(self._head)
-        expr.elements = self._elements
+        expr._elements = self._elements
```

Comment:
this could be put in the previous line as 
expr = Expression(self._head, self._elements, element_properties={"_please_do_not_convert_me"=True,})

Reply:
RIght now, I want to defer this for another some future iteration.